frontend/app/views/tier/change.scala.html

```diff
@@ -2,37 +2,76 @@
-                    }
+@main("Change Tier") {
+
+    <main role="main" class="page-content l-constrained">
+
+        @fragments.page.pageHeader("Change tier")
 
-                </section>
+        <section class="page-section">
+            <div class="page-section__lead-in">
+                <a href="@Config.idWebAppUrl/membership/edit" class="action action--secondary u-anchor u-no-top-margin">
+                    @fragments.actionIcon("arrow-left", leftIcon=true)
+                    <span class="action__label">Edit profile</span>
+                </a>
             </div>
-            <div class="content__container content__container--border-dark content__footer">
+            <div class="page-section__content">
+
+                <h1 class="h-section">Choose the tier you would like to switch to:</h1>
+
+                @* Packages *@
+                <ul class="grid grid--bordered grid--stretch grid--3up-stacked">
+                    @for(tier <- Tier.allPublic) {
+                        <li class="grid__item">
 
-                <ul class="inline-list inline-list--divided">
+                            @fragments.tier.packageChanger(tier, currentTier)
+
+                            @* Benefits *@
+                            <div class="benefits-chooser">
+                                <a class="mini-link until-desktop js-toggle" data-toggle="toggle-benefit-@tier.toString.toLowerCase" data-toggle-label="Hide what's included" data-toggle-icon="arrow-up">
+                                    <span class="toggle-link__icon icon-holder icon-holder--small">
+                                        @fragments.inlineIcon("arrow-down", List("inline-icon--small"))
+                                    </span>
+                                    <span class="js-toggle-label">Show what's included</span>
+                                </a>
+                                <div class="benefits-chooser__list" id="toggle-benefit-@tier.toString.toLowerCase">
+                                    <ul class="list-stack">
+                                        <li class="list-stack__title">
+                                            What's included
+                                        </li>
+                                        @for(benefit <- Benefits.details(tier).list) {
+                                            <li>@fragments.benefits.item(benefit)</li>
+                                        }
+                                    </ul>
+
+                                    @if(tier != currentTier) {
+                                        <a class="action"
+                                           href="@routes.TierController.upgrade(tier)"
+                                           data-metric-trigger="click"
+                                           data-metric-category="changeTier"
+                                           data-metric-action="@tier.toString.toLowerCase()"
+                                        >
```

Comment:
could we add a hook here please? something like qa-confirm which is tier-independent.